src/repository/AliasStore.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from "fs-extra";
+import { CREATE_PATH } from "../const";
+import * as path from "path";
+import { InvalidArgumentError } from "../errors/InvalidArgumentError";
+import { IAlias, IAliases } from "../types";
+
+export class AliasStore {
+  static readonly ALIAS_FILE = path.join("jira", "alias.json");
```

Comment:
Do you use the 'ALIAS_FILE'-Variable somewhere else or why is it in here? I would like to keep things consistent (like in JiraStore as consts) but if you got an good argument i'd also prefer your way :)

Reply:
No good arguments - just old habit.
I will update the path to not pollute the `jira` directory and refactor it to a const variable.